dev/tidb-in-kubernetes/reference/configuration/local-pv.md

```diff
@@ -15,3 +15,15 @@ Kubernetes currently supports statically allocated local storage. To create a lo
 2. Install the `local-volume-provisioner` program. See the [Helm installation procedure](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/helm) for reference.
 
 For more information, refer to [Kubernetes local storage](https://kubernetes.io/docs/concepts/storage/volumes/#local) and [local-static-provisioner document](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner#overview).
+
+## Data Security
+
+By default, when a local PV is released, the provisioner recycles it. You can set the reclaim policy of your storage class to `Retain` to prevent data from being recycled automatically. After confirming that a PV's data can be deleted, modify its reclaim policy to `Delete`. For how to change PV reclaim policy in Kubernetes, refer to [this doc](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/).
```

Comment:
```suggestion
By default, when a local PV is released, the provisioner recycles it. To prevent data from being recycled automatically, you must set the reclaim policy of your storage class to `Retain`. After confirming that a PV's data can be deleted, modify its reclaim policy to `Delete`. For how to change PV reclaim policy in Kubernetes, refer to [this doc](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/).
```